job_spec/WATER_MAP.yml

```
@@ -83,6 +110,16 @@ WATER_MAP:
     - Ref::hand_fraction
     - --membership-threshold
     - Ref::membership_threshold
+    - --include_flood_depth
+    - Ref::include_flood_depth
+    - --estimator
+    - Ref::estimator
```

Continued from above:
```suggestion
    - Ref::flood_depth_estimator
```